src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -254,7 +254,7 @@ public void Delete(int id)
 
             if (game == null)
             {
-                throw new ArgumentNullException("game");
+                throw new ArgumentNullException($"game");
```

Comment:
?

Reply:
same thing

Reply:
this way you basically fooling checker. throw invalid argument exception with id